pkg/tests/handlers/accounts_test.go

```diff
@@ -0,0 +1,101 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cloud-bulldozer/ocm-api-load/pkg/logging"
+	"github.com/cloud-bulldozer/ocm-api-load/pkg/types"
+)
+
+func Test_clusterAuthorizationsBody(t *testing.T) {
+	type args struct {
+		ctx       context.Context
+		clusterID string
+		options   *types.TestOptions
+	}
+	logger, _ := logging.NewGoLoggerBuilder().Build()
+	tests := []struct {
+		name string
+		args args
+	}{
+
+		{"randomize-1", args{context.TODO(), "clusterID-1", &types.TestOptions{Logger: logger}}},
+		{"randomize-2", args{context.TODO(), "clusterID-2", &types.TestOptions{Logger: logger}}},
+		{"randomize-3", args{context.TODO(), "clusterID-3", &types.TestOptions{Logger: logger}}},
+		{"randomize-4", args{context.TODO(), "clusterID-4", &types.TestOptions{Logger: logger}}},
+		{"randomize-5", args{context.TODO(), "clusterID-5", &types.TestOptions{Logger: logger}}},
+		{"randomize-6", args{context.TODO(), "clusterID-6", &types.TestOptions{Logger: logger}}},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := clusterAuthorizationsBody(tt.args.ctx, tt.args.clusterID, tt.args.options)
+			if got == nil {
+				t.Errorf("clusterAuthorizationsBody() = %v", got)
+			} else {
+				if !strings.Contains(string(got), tt.args.clusterID) {
+					t.Errorf("Clusterid not in body = %v", string(got))
+				}
+			}
+		})
+	}
+}
+
+func Test_randomizeCount(t *testing.T) {
+	rand.Seed(time.Now().UnixNano())
+	tests := []struct {
+		name     string
+		mincount int
+		maxcount int
+	}{
+		{"Testing range of random count generated-1", 4, 10},
+		{"Testing range of random count generated-2", 4, 10},
+		{"Testing range of random count generated-3", 4, 10},
+		{"Testing range of random count generated-4", 4, 10},
+		{"Testing range of random count generated-5", 4, 10},
+		{"Testing range of random count generated-6", 4, 10},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := randomizeCount()
+			if got >= tt.mincount && got <= tt.maxcount {
+				fmt.Printf("Count %d", got)
+				// t.Errorf("randomizeCount() = %v", got)
+			}
+		})
+	}
+}
+
+func Test_randomizeResourceName(t *testing.T) {
+	rand.Seed(time.Now().UnixNano())
+	tests := []struct {
+		name             string
+		resourcenamelist [5]string
+	}{
+		{"Testing to check if output returned is from AWSResourcename-1", [5]string{"m3.2xlarge", "m4.large", "m4.xlarge", "m5.large", "m5.xlarge"}},
```

Comment:
You are using the same slice 5 times, you can use a variable for that, or even use the helpers values.